### ISSUE TITLE
How much will the loss function reduce after the model converges?

### ISSUE THREAD

**Issue by hustCYQ** [Tianxiaomo/pytorch-YOLOv4#499]
![image](https://user-images.githubusercontent.com/39451786/147396843-f0a002e6-b128-486c-8fd0-c8bdc0b7730c.png)

I found that after 10 epochs, the model almost converged, but the loss_xy did not decrease. I don’t know if the loss will decrease again if I continue to wait.

**Comment by DHolmanCoding** [comment]
I also observed the XY and WH loss not decreasing at all
